StudyPathTraker/StudyPathTraker/Extensions/UINavigationController+Extensions.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UINavigationController+Extensions.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 5/30/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import UIKit
+
+extension UINavigationController {
+
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+        self.delegate = self as? UINavigationControllerDelegate
```

Comment:
this works? I don't remember that UINavigationController implements itself this delegate.